docs/admin/config/batch_changes.mdx

```diff
@@ -371,16 +396,12 @@ batchSpecLibrary(first: 100) {
 }
 ```
 
-### Featured Templates
-
-<Callout type="note">Featured templates are supported in Sourcegraph v6.4 and more.</Callout>
+### Featured Records
 
-Site-admins can mark a template as featured by either clicking the star button next to the list of library records. Featured records will automatically move to a section atop the remaining library records.
+Site-admins can mark a record as featured by either clicking the star button next to the list of library records. Featured records will automatically move to a section atop the remaining library records. This will add or remove the `"featured"` [label](#labels).
```

Comment:
This seems unfinished because you use "by either..." but then only list 1 option: "Site-admins can mark a record as featured by either clicking the star button next to the list of library records." Did you mean to list another?

Reply:
I think we can remove "This will add or remove the `"featured"` [label](#labels)." It seems self-explanatory IMO